applications/StructuralMechanicsApplication/custom_elements/cr_beam_element_linear_3D2N.cpp

```diff
@@ -107,6 +107,36 @@ void CrBeamElementLinear3D2N::CalculateLeftHandSide(
   KRATOS_CATCH("")
 }
 
+void CrBeamElementLinear3D2N::CalculateMassMatrix(MatrixType &rMassMatrix,

```

Comment:
Strange, it does not heritage the function from the base?

Reply:
no it needs always the original system as transformation in the linear case. In the bass class I am using an updated transformation which is not initialzied in the linear element. Thus I need this change to make it work

Reply:
This was already happening before the condensation branch was merged where I started inheriting the elements